libs/sdk-common/src/lnurl/specs/pay.rs

```diff
@@ -132,6 +132,11 @@ pub mod model {
         pub data: LnUrlPayRequestData,
         /// The amount in millisatoshis for this payment
         pub amount_msat: u64,
+        /// Trampoline payments outsource pathfinding to the LSP. Trampoline payments can improve
+        /// payment performance, but are generally more expensive in terms of fees and they
+        /// compromise on privacy. Setting this flag to `true` will ensure no trampoline payment is
+        /// attempted.
```

Comment:
Guess because of the rename
```suggestion
        /// compromise on privacy. Setting this flag to `false` will ensure no trampoline payment is
        /// attempted.
```

Reply:
Thanks, I removed that sentence, it's overly verbose now this is a 'positive' boolean.